hr_payroll_expense/data/hr.salary.rule.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+File: hr.salary.rule.xml
+Module: hr_payroll_expense
+
+Created by dco@open-net.ch
+
+Copyright (c) 2014 Open-Net Ltd.
+-->
```

Comment:
@dufresnedavid following https://github.com/OCA/maintainer-tools/blob/master/template/module/views/assets.xml and https://github.com/OCA/maintainer-tools/blob/master/template/module/data/model_name_data.xml, yes it is advised on data and views.
For data, I reckon that it is awkward sometimes for data.
To be noted that in this module, the copyright should be corrected to follow OCA templates